homeassistant/components/tessie/binary_sensor.py

```diff
@@ -24,6 +24,8 @@ class TessieBinarySensorEntityDescription(BinarySensorEntityDescription):
     """Describes Tessie binary sensor entity."""
 
     is_on: Callable[..., bool] = lambda x: x
+    frozen = True
+    kw_only = True
```

Comment:
This is the wrong place, it should be set in the dataclass decorator like `@dataclass(kw_only=True, frozen=True)`. AFAIK `frozen=True` will not work at the moment because the necessary PR (#105512) for the base EntityDescriptions (e.g. `BinarySensorEntityDescription`) has not been merged yet.

Reply:
That would make way more sense. I'll get it fixed up.